LDM-533.tex

```diff
@@ -231,6 +236,8 @@ \subsection{Suspension criteria and resumption requirements}
 
 \subsection{Naming convention}
 
+{\it This naming convention is now obsolete, since the unique identifier will be the JIRA test case ID.}
```

Comment:
OK. Need to be reworded. LDM-639 has already a definition. It may be worth mentioning that that the old test cases already written and approved, will still have that identifier at the beginning of the name.